nautobot/docs/release-notes/version-1.0.md

```diff
@@ -4,15 +4,170 @@ This document describes all new features and changes in Nautobot 1.0, a divergen
+
+Job result history is now retained indefinitely unless intentionally deleted. Historically only the most recent result for each custom script or report was retained and all older records were deleted.
+
+Python modules that define jobs can now be stored in Git and easily added to Nautobot via the UI as documented above in [Git Integration as a Data Source](#git-integration-as-a-data-source).
+
+#### Custom User Model
+
+A new custom model has been created for `User` data. This has allowed Nautobot to use a UUID as a primary key for the `User` model, and to prepare for future use-cases not support by the default Django model.
+
+This has also meant `UserConfig` no longer exists as a separate model. `UserConfig` is now a property on the custom `User` class.
+
+#### Hiding UI Elements based on Permissions
+
+Historically, a user viewing the home page and navigation menu would see a list of all model types and menu items in the system, with a “lock” icon on items that they were not granted access to view in detail.
+
+As an [option](../configuration/optional-settings.md#hide_restricted_ui), administrators can now choose to instead hide un-permitted items altogether from the home page and the navigation menu, providing a simpler interface for limited-access users. The prior behavior remains as the default.
+
+#### IPAM Network Fields to VARBINARY
+
+For the support of databases other than Postgres, the network fields inside of IPAM needed to be changed. `cidr` and `inet` field types have been replaced with a database agnostic field type. For this purpose `varbinary` was chosen because it can safely and efficiently store packed binary integers.
+
+#### Navigation Menu Changes
+
+The "Other" menu has been renamed to "Extensibility" and many new items have been added to this menu.
+
+[Status](../models/extras/status.md) records have been added to the "Organization" menu.
+
+#### New Name and Logo
+
+NetBox has been changed to Nautobot throughout the code, UI, and documentation, and Nautobot has a new logo.
+
+#### User-Defined Custom Links
+
+Nautobot allows for the definition of [custom links](../models/extras/customlink.md) to add to various built-in data views. These can be used to provide convenient cross-references to other data sources outside Nautobot, among many other possibilities.
+
+Historically this feature was restricted so that only administrators could define and manage custom links. In Nautobot this has been moved into the main user interface, accessible to any user who has been granted appropriate access permissions.
```

Comment:
```suggestion
Historically the [custom links](../models/extras/customlink.md) feature was restricted so that only administrators could define and manage custom links to add to various built-in data views. In Nautobot the management of custom links has been moved into the main user interface, accessible to any user who has been granted appropriate access permissions.
```